mirdata/datasets/tiny_4mula.py

```diff
@@ -0,0 +1,319 @@
+import os
+from mirdata.download_utils import RemoteFileMetadata, downloader
+from mirdata import core, jams_utils
+from numpy import load, ndarray
+from pandas import read_csv, read_parquet
+
+
+BIBTEX = """inproceedings{10.1145/3428658.3431089,
+  author = {da Silva, Angelo Cesar Mendes and Silva, Diego Furtado and Marcacini, Ricardo Marcondes},
+  title = {4MuLA: A Multitask, Multimodal, and Multilingual Dataset of Music Lyrics and Audio Features},
+  year = {2020},
+  isbn = {9781450381963},
+  publisher = {Association for Computing Machinery},
+  address = {New York, NY, USA},
+  url = {https://doi.org/10.1145/3428658.3431089},
+  doi = {10.1145/3428658.3431089},    
+  booktitle = {Proceedings of the Brazilian Symposium on Multimedia and the Web},
+  pages = {145–148},
+  numpages = {4},
+  keywords = {latin musical dataset, music dataset, multimodal musical dataset},
+  location = {S\~{a}o Lu\'{\i}s, Brazil},
+  series = {WebMedia '20}
+}"""
+
+REMOTES = {
+    "dataset": RemoteFileMetadata(
+        filename="4mula_tiny.parquet",
+        url="https://zenodo.org/record/4585498/files/4mula_tiny.parquet?download=1",
+        checksum="b1caba387baa22a7b16893702a3214f7",
+        destination_dir=".",
+    )
+}
+
+LICENSE_INFO = "Creative Commons Attribution 4.0 International."
+
+DOWNLOAD_INFO = """
+  This loader is designed to load the melspectrogram and metadata, as it is available for download.
+  Unfortunately the audio files of the 4MULA dataset are not available
+  for download. You can more details and follow the updates in our repository: 
+  ==> https://github.com/4mulaDataset/4mula    
+"""
+
+DEFAULT_COLUMNS = [
+    "music_id",
+    "music_name",
+    "music_lang",
+    "music_lyrics",
+    "art_id",
+    "art_name",
+    "art_rank",
+    "main_genre",
+    "related_genre",
+    "related_art",
+    "related_music",
+    "musicnn_tags",
+    "melspectrogram",
+]
+
+
+class Track(core.Track):
+    """tiny_4mula track class
+
+    Args:
+        track_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets/tiny_4mula`
+
+    Attributes:
+        music_id (str): track id
+        music_name (str): music name
+        music_lang (str): music language
+        music_lyrics (str): lyrics
+        art_id (str): artist id
+        art_name (str): artist name
+        art_rank (int): artist rank
+        main_genre (str): artist main genre
+        related_genre (list): artist related genre
+        related_art (list): others artist related to artist
+        related_music (list): others music related to track
+        musicnn_tags (list): top-3 tags got by Musicnn
+        melspectrogram (numpy.ndarray): melspectrogram extracted by librosa
+
+
+    Cached Properties:
+        melspectrogram: melspectrogram in npy file
+        annotation: csv file with annotated metadata (all attributes, except melspectrogram)
```

Comment:
I'm a bit confused about this cached property, it seems is redundant with the track properties and I don't see it in the code. Maybe remove?